src/connection.tsx

```diff
@@ -361,6 +381,20 @@ function composeAuthentication(connection: ConnectionOptions): AnyAuth | Token |
 	}
 }
 
+function mapResults(response: QueryResult<unknown>[]): QueryResponse[] {
+	return response.map(res => {
+		return res.status == "OK" ? {
```

Comment:
nitpick: Might just do this if it is the sole difference

```suggestion
         return {
            success: res.status === "OK",
			result: res.result,
			execution_time: res.time
	      };
```